CDP4Authentication/Interfaces/IAuthenticatorProperties.cs

```diff
@@ -33,5 +33,10 @@ public interface IAuthenticatorProperties
         /// Gets or sets the description.
         /// </summary>
         string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the server salts(WSP specific).
+        /// </summary>
+        string[] ServerSalts { get; set; }
```

Comment:
we are adding a WSP authentication property to the generic authentication properties. I doubt this is the appropriate approach.

Reply:
I will add a specific interface for the WSP authentication property.